jib-core/src/main/java/com/google/cloud/tools/jib/http/Request.java

```diff
@@ -50,9 +50,7 @@ public Request build() {
      * @return this
      */
     public Builder setAuthorization(@Nullable Authorization authorization) {
-      if (authorization != null) {
-        headers.setAuthorization(authorization.toString());
-      }
+      headers.setAuthorization(authorization == null ? null : authorization.toString());
```

Comment:
Probably intended, but this changes the behavior (if header's authorization is not null, setting `null` in this case sets header's authorization to null, but with the old behavior setting `null` would leave it unchanged). Is this fine?

Reply:
Good question. It changes the behavior, but there's no code where we call this more than once, so we are safe. And I think the new behavior makes more sense.